Mathlib/RingTheory/Nilpotent/Exp.lean

```diff
@@ -164,4 +165,56 @@ theorem exp_of_nilpotent_is_unit {a : A} (h : IsNilpotent a) : IsUnit (exp a) :=
   refine ⟨exp (-a), h₃.symm, ?_⟩
   rw [← exp_add_of_commute h₁.symm h₂ h, neg_add_cancel a, exp_zero_eq_one]
 
+theorem map_exp {B F : Type*} [Ring B] [FunLike F A B] [RingHomClass F A B] [Module ℚ B]
```

Comment:
Could you add this one too?

https://github.com/leanprover-community/mathlib4/blob/0e29a68d13815c89d3b4f810ccf261d02afd3e84/Mathlib/Analysis/Normed/Algebra/Exponential.lean#L505-L507